telemetry/tokens.go

```diff
@@ -0,0 +1,58 @@
+package telemetry
+
+import (
+	cosmostelemetry "github.com/cosmos/cosmos-sdk/telemetry"
+	"github.com/hashicorp/go-metrics"
+)
+
```

Comment:
// TODO_MAINNET(@bryanchriswhite): Revisit how telemetry is managed under `x/tokenomics` to ensure that it
// complies with the new hardened settlement approach.

Reply:
added